src/panels/config/dashboard/ha-config-dashboard.js

```diff
@@ -47,6 +48,10 @@ class HaConfigDashboard extends LocalizeMixin(PolymerElement) {
             <ha-config-entries-menu hass="[[hass]]"></ha-config-entries-menu>
           </template>
 
+          <template is="dom-if" if="[[computeIsLoaded(hass, &quot;config.auth_provider_homeassistant&quot;)]]">
```

Comment:
maybe you can use `'` for better readability

Reply:
Yeah that's a weird artifact from the modulizer.